ts/smelter-web-wasm/src/compositor/compositor.ts

```diff
@@ -36,7 +36,16 @@ export default class Smelter {
   private coreSmelter?: CoreSmelter;
   private instance?: WasmInstance;
   private options: SmelterOptions;
-  private logger: Logger = pino({ level: 'warn' });
+  private logger: Logger = pino({
+    level: 'warn',
+    browser: {
+      asObject: true,
+      write: {
+        debug: console.log,
```

Comment:
instead of console.trace that generates a  lot of output that makes it hard to read